src/pages/Appearance/Datasets/DialogDatasetSpecificSetting.js

```diff
@@ -37,7 +37,7 @@ const DialogDatasetSpecificSetting = ({
                     ) : (
                         <Select
                             name="id"
-                            inputWidth="300px"
+                            inputWidth="350px"
```

Comment:
Hey @Sharmyn28 , 

Does it make sense to use the `dhis2/ui` `spacers` constant here? Something like this:
 `inputWidth={spacers.dp384}`. 

Thanks!